src/Generator/Base/ClassEntity.cs

```diff
@@ -24,6 +24,7 @@ protected override void WriteUsingStatements(StreamWriter streamWriter)
         WriteUsingAzure(streamWriter);
         WriteUsingAdt(streamWriter);
         WriteUsingSystem(streamWriter);
+        WriteUsingSystemRuntime(streamWriter);
```

Comment:
Is this necessary? The [DateOnly documentation](https://learn.microsoft.com/en-us/dotnet/api/system.dateonly?view=net-6.0) has the struct in the `System` namespace, not `System.Runtime`.